front_matter/_formats.ts

```diff
@@ -21,9 +21,7 @@ const JSON_HEADER = `(---json|${JSON_DELIMITER})\\s*`;
 const JSON_FOOTER = `(?:---|${JSON_DELIMITER})`;
 
 const DATA = "([\\s\\S]+?)";
-const NEWLINE = `${
-  globalThis?.Deno?.build?.os === "windows" ? "\\r?" : ""
-}(?:\\n)?`;
+const NEWLINE = "\\r?\\n?";
```

Comment:
```suggestion
const NEWLINE = "\\r?(?:\\n)?";
```

Reply:
I think these brackets don't do anything.

Reply:
Let's merge as-is but keep this suggestion in mind in case anyone runs into any issues.